src/Sulu/Bundle/WebsiteBundle/Tests/Application/Kernel.php

```diff
@@ -48,7 +48,7 @@ public function registerContainerConfiguration(LoaderInterface $loader): void
 
     public function registerBundles(): iterable
     {
-        $bundles = parent::registerBundles();
+        $bundles = [...parent::registerBundles()];
```

Comment:
is typed as `iterable` to push another bundle we need to convert the iterable to array.